### DISCUSSION TITLE
Running Cadmium as a live system from USB?

### DISCUSSION THREAD

**Discussion by wehkah** [Maccraft123/Cadmium#138] [General]
Hello!

I am desperate seeking to replace ChromeOS on my Duet with Linux. However, I would like to give it a good try before I do so, to see if my Chromebook will swallow it. Is it possible to run Cadmium as a live system from USB for a short time without installing it?

**Comment by chinh4** [comment]
From what I know there currently isn't any live graphical user interface with cadmium, the only thing available is a cli.

**Comment by Maccraft123** [comment]
In 0.3.0 release there is `install-to-here` and in 0.4.0-pre2 the installer
will ask you

niedz., 31 lip 2022, 19:20 użytkownik chinh4thepro ***@***.***>
napisał:

**Comment by wehkah** [comment]
Thanks for replying!

I managed to make my CB boot from USB and I started the installer script. So, when I choose "here", it will install the hole Cadmium system to the USB drive, and the system can be run as a live system?

That sounds promising!! :)

~~I cannot make the wifi working for the installer: I choose the right network, I enter the password, and my router tells me that the CB is connected to the wifi. But the GUI (ncurses) only offers two buttons after entering the password: "Deactivate" and "Quit". There's not button saying anything like "continue". Since I don't want to deactivate the wifi, I choose <Quit>, but then the installer says it can't reach the repo server. 

What am I doing wrong there?~~


EDIT: I got it! After the installer breaks because it can't find the files to download, you have to do a "apt-get update", then start the installer again. After quitting the wifi setup, the installer goes on with its work.

I'd suggest to make "apt-get update" part of the installer script.

**Comment by Maccraft123** [comment]
Yeah you just select `Quit` option

niedz., 31 lip 2022 o 19:42 Wolfgang Klein ***@***.***>
napisał(a):
